content/influxdb/v2.0/query-data/common-queries/compare_last_measurement.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Compare the last measurement to another bucket's mean
+seotitle: Compare the last measurement to a mean stored in another bucket
+description: >
+  This example is useful for writing a mean to a bucket and using it as a threshold check.
+influxdb/v2.0/tags: [queries]
+menu:
+  influxdb_2_0:
+    name: Compare the last measurement to another bucket's mean
+    parent: Common queries
+weight: 104
+---
+
+
+This example is useful for writing a mean to a bucket and using it as a threshold check. It compares the last measurement to a mean stored in another bucket by doing the following:
```

Comment:
Compare the value from the latest point to an average value stored in another bucket. This is useful when using the average value to calculate a threshold check.